requirements.txt

```diff
@@ -3,6 +3,9 @@ django
 djangorestframework
 # cors headers for SPA apps to support csrf
 django-cors-headers
+# rendering open API doc
+pyyaml
+coreapi
 # static file serving
 whitenoise
 markdown
```

Comment:
not that important, but good practice is to include the package versions, otherwise this is always installing the latest version of the package, which might break the app when we stop maintaining it in the future.

Reply:
good point. I would leave this as is since python is not crazy like npm and we probably won't have issues in near future (a year or something)